main.c

```diff
@@ -78,12 +78,12 @@ int wait_ESP32(UART * pUART, UINT24 baudRate) {
 	open_UART0(pUART);					// Open the UART 
 	init_timer0(10, 16, 0x00);  		// 10ms timer for delay
 	gp = 0;								// Reset the general poll byte	
-	for(t = 0; t < 20; t++) {			// A timeout loop (20 x 50ms = 1s)
+	for(t = 0; t < 20; t++) {			// A timeout loop (20 x 100ms = 2s)

```

Comment:
The lower baud rate was left in as a fallback when both VDP and MOS ran at 384000. I'll leave it in for the moment in case there are any legacy issues, and will just lengthen the timeout as suggested by #89